ofono/src/sms.c

```diff
@@ -1174,6 +1296,13 @@ static const GDBusMethodTable sms_manager_methods[] = {
 			GDBUS_ARGS({ "to", "s" }, { "text", "s" }),
 			GDBUS_ARGS({ "path", "o" }),
 			sms_send_message) },
+	{ GDBUS_ASYNC_METHOD("SendDataMessage",
+			GDBUS_ARGS(
+				{ "to", "s" },
+				{ "text", "s" },
```

Comment:
I'm not entirely sure what kind of data can a data message include, but to me "ay" would sound better for a data blob.

Reply:
Yes but in case of AML and the 7bit encoding already implemented here I don't really see the benefit of implementing the encoding of the message outside Ofono. After all, AML is the main use here.